src/core/math/distributions/DistributionExponentialError.cpp

```diff
@@ -21,6 +23,28 @@
 
 using namespace RevBayesCore;
 
+/*!
```

Comment:
Wouldn't it make sense to add this function to StringUtilities?

Reply:
Great point – moved it there!